sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -351,8 +387,12 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 
 [detect_language_sample]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample1_DetectLanguage.md
 [analyze_sentiment_sample]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2_AnalyzeSentiment.md
+[analyze_sentiment_opinion_mining_sample]: https://github.com/maririos/azure-sdk-for-net/blob/cd74dde9546ea675f9289e1b4e6fd804bda2a3bc/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2.1_AnalyzeSentimentWithOpinionMining.md
+<!--[analyze_sentiment_opinion_mining_sample]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample2.1_AnalyzeSentimentWithOpinionMining.md-->
```

Comment:
question: what is the point of this commented out part?

Reply:
To make CI pass the links need to work, so I added links pointing to this commit but have commented the master links so once this is merged, I can go and remove all commented links and remove the static commit link

Reply:
why you are not referring to the file on master?

Reply:
.NET repo has a CI step that will check that all links work.
Because the file didn't exist in master until the PR is merged, I added the commit to the link and commented the link to master.
In this PR https://github.com/Azure/azure-sdk-for-net/pull/14871 I am removing those links and leaving only the master ones